API/Data/Repositories/ReadingListRepository.cs

```diff
@@ -218,6 +218,7 @@ public async Task<IEnumerable<ReadingListItemDto>> GetReadingListItemDtosByIdAsy
             if (progressItem == null) continue;
 
             progressItem.PagesRead = progress.PagesRead;
+            progressItem.ProgressLastModifiedUtc = progress.LastModifiedUtc;
```

Comment:
Add the non Utc field as well to keep consistent.

Reply:
ok